doc/conf.py

```diff
@@ -35,7 +35,7 @@
 # ones.
 
 sys.path.insert(0, os.path.join(os.path.abspath('.'), 'extensions'))
-extensions = ['breathe', 'sphinx.ext.graphviz', 'sphinx.ext.extlinks', 'kerneldoc']
+extensions = ['breathe', 'sphinx.ext.graphviz', 'sphinx.ext.extlinks']
```

Comment:
Why did you remove the `kerneldoc` extension here?

I'm getting a bunch of errors when trying to build the documentation because of this:
```
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:34: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: drivers/gpu/drm/i915/i915_vgpu.c
   :doc: Intel GVT-g guest support
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:37: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: drivers/gpu/drm/i915/i915_vgpu.c
   :internal:
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:43: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: drivers/gpu/drm/i915/intel_gvt.c
   :doc: Intel GVT-g host support
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:46: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: drivers/gpu/drm/i915/intel_gvt.c
   :internal:
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:67: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: include/linux/vhm/vhm_vm_mngt.h
   :functions: put_vm
               vhm_get_vm_info
               vhm_inject_msi
               vhm_vm_gpa2hpa
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:73: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: include/linux/vhm/acrn_vhm_ioreq.h
   :internal:
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:76: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: include/linux/vhm/acrn_vhm_mm.h
   :functions: acrn_hpa2gpa
               map_guest_phys
               unmap_guest_phys
               add_memory_region
               del_memory_region
               write_protect_page
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:127: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: drivers/gpu/drm/i915/gvt/mpt.h
   :functions: intel_gvt_hypervisor_host_init
               intel_gvt_hypervisor_host_exit
               intel_gvt_hypervisor_attach_vgpu
               intel_gvt_hypervisor_detach_vgpu
               intel_gvt_hypervisor_inject_msi
               intel_gvt_hypervisor_virt_to_mfn
               intel_gvt_hypervisor_enable_page_track
               intel_gvt_hypervisor_disable_page_track
               intel_gvt_hypervisor_read_gpa
               intel_gvt_hypervisor_write_gpa
               intel_gvt_hypervisor_gfn_to_mfn
               intel_gvt_hypervisor_map_gfn_to_mfn
               intel_gvt_hypervisor_dma_map_guest_page
               intel_gvt_hypervisor_dma_unmap_guest_page
               intel_gvt_hypervisor_set_trap_area
               intel_gvt_hypervisor_set_pvmmio
               intel_gvt_hypervisor_dom0_ready
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:172: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: drivers/gpu/drm/i915/gvt/cfg_space.c
   :functions: intel_vgpu_emulate_cfg_read
               intel_vgpu_emulate_cfg_write
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:176: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: drivers/gpu/drm/i915/gvt/mmio.c
   :functions: intel_vgpu_emulate_mmio_read
               intel_vgpu_emulate_mmio_write
/home/gvancuts/work/acrn/acrn-hypervisor/doc/api/GVT-g_api.rst:180: WARNING: Unknown directive type "kernel-doc".

.. kernel-doc:: drivers/gpu/drm/i915/gvt/vgpu.c
   :functions: intel_gvt_create_vgpn
               intel_gvt_destroy_vgpu
               intel_gvt_reset_vgpu
               intel_gvt_activate_vgpu
               intel_gvt_deactivate_vgpu

```